### PR TITLE
Add a shortcut which jumps to top by Ctrl-G

### PR DESCRIPTION
**Pull request by asmsuechan**
This PR adds a shortcut which jumps to a top of notes by Ctrl+G. Any ideas for the shortcut?

### CODE REVIEW

**Comment by asmsuechan** [comment]
screen shot 2017-02-18 at 17 31 58

File: browser/main/NoteList/index.js [hunk 1]
```diff
@@ -324,6 +330,23 @@ class NoteList extends React.Component {
...
   }
 
+  jumpToTop() {
+    if (this.notes == null || this.notes.length === 0) {
+      return
+    }
+    let { router } = this.context
+    let { location } = this.props
+
+    let targetIndex = 0
```
Comment (sota1235) [inline browser/main/NoteList/index.js]: Plz use `const` :sob:

**Comment by sota1235** [comment]
That's all from me.

> @kazup01 
Please check the function on Windows.

File: browser/main/NoteList/index.js (hunk 1 above)
Comment (asmsuechan) [inline browser/main/NoteList/index.js]: Alright

**Comment by kazup01** [comment]
@sota1235 Since it is not in hurry, can I check on Saturday?

File: browser/main/NoteList/index.js [hunk 2]
```diff
@@ -324,6 +330,23 @@ class NoteList extends React.Component {
     }
   }
 
+  jumpToTop() {
+    if (this.notes == null || this.notes.length === 0) {
```
Comment (sota1235) [inline browser/main/NoteList/index.js]: Can you use `===`? 
Or you can use `_.isNull` and `_.isUndefined`.

**Comment by sota1235** [comment]
Please resolve conflicts 😭

File: browser/main/NoteList/index.js (hunk 2 above)
Comment (asmsuechan) [inline browser/main/NoteList/index.js]: Yes. But I don't know what the difference between `===` and `_.isNull` thus I used `===` because I knew it.

**Comment by asmsuechan** [comment]
Alright. Done.

File: browser/main/NoteList/index.js [hunk 3]
```diff
@@ -51,7 +55,8 @@ class NoteList extends React.Component {
     ee.on('list:next', this.selectNextNoteHandler)
     ee.on('list:prior', this.selectPriorNoteHandler)
     ee.on('list:focus', this.focusHandler)
-    ee.on('list:isMarkdownNote', this.alertIfSnippetHandler)
+    ee.on('list:isMarkdownNote', this.alertIfSnippetHnalder)
```
Comment (sota1235) [inline browser/main/NoteList/index.js]: 😇 😇 😇 😇 😇 😇 

image

Comment (asmsuechan) [inline browser/main/NoteList/index.js]: Sorry. Fixed.

**Comment by sota1235** [comment]
I can confirm on windows. Thanks a lot for fixing!
LGTM

**Pull request merged**